agent/agents/supervisor/supervisor.go

```diff
@@ -672,6 +680,33 @@ func (s *Supervisor) processParams(agentID string, agentProcess *agentpb.SetStat
 	return &processParams, nil
 }
 
+func (s *Supervisor) version(agentType inventorypb.AgentType, path string) (string, error) {
```

Comment:
TODO: check pmm version before call CLI --version

Reply:
It's code will be available in 2.33 so skip this check, but save in comments, just in case
```go
	pmmAgentVersion, err := version.Parse(version.Version)
	if err != nil {
		return "", errors.Wrapf(err, "failed to parse PMM agent version %q", version.Version)
	}

	if pmmAgentVersion.Less(exporterCommandVersion) {
		return "", nil
	}
```